lib/salus/scanners.rb

```diff
@@ -1,6 +1,7 @@
 module Salus::Scanners; end
 
-Dir.entries(File.expand_path('scanners', __dir__)).each do |filename|
+# Sort to avoid race coniditions in what order scanners are loaded.
+Dir.entries(File.expand_path('scanners', __dir__)).sort.each do |filename|
```

Comment:
how did you determine sorting as the issue here?

Reply:
This block of code dictates the order in which the various scanner classes are loaded.  Ruby doesn't guarantee the order in which Dir.entries will return files.

Manually placing cargo_audit.rb as the first entry reproduced the production error. 

By sorting, we are removing the varying order in which classes may load - and the race conditions that can occur if there is any order dependence.  This ensures that as we run executions across our various pipelines (dev, testing, production) we will remove discrepancies in the load process.  The sort itself doesn't matter rather it's just a way to ensure consistent loading between various environments.